tools/TreeBest/TreeBest_best_wrapper.xml

```diff
@@ -0,0 +1,121 @@
+<tool id="treebest_best_wrapper" name="TreeBest best" version="1.9.2">
+    <description>
+        Generate a Phylogeny tree using CDS alignment and species tree
+    </description>
+     <requirements>
+        <requirement type="package" version="1.9.2">treebest</requirement>
+    </requirements>
+    <command>
+<![CDATA[
+treebest best 
+-f 
+$species_tree
+$P
+$S
+$A
+#if $C
+-C $C
+#end if
+$s
+$g
+$N
+-c $c
+-d $d
+-l $l
+-L $L
+-b $b
+#if $K.K_selector == 'yes'
+-k e 
+#else 
+-k $K.k
+#end if
+
+#if $A_option.A_selector == 'yes'
+-a e 
+#else 
+-a $A_option.a
+#end if
+
+-F $F
+$aln
+-p "Galaxy_TreeBest_Best"
+> $output
+]]>
+    </command>
+
+    <inputs>
+        <param name="species_tree" type="data" format="nhx" label="Species file in Newick format" help="-f"/>
+        
+        <param name="aln" type="data" format="fasta" label="CDS alignment"/>
+        
+        <param name="P" type="boolean" truevalue="-P" falsevalue="" label="Skip PHYML" help="(-P)" />
+         
+        <param name="S" type="boolean" truevalue="-S" falsevalue="" label="Ignore the prob. of gene evolution (NOT recommended)" help="(-S)" />
+
+        <param name="A" type="boolean" truevalue="-A" falsevalue="" label="Apply constraint to PHYML" help="(-A)" />
+    
+        <param name="r" type="boolean" truevalue="-r" falsevalue="" label="Discard species that do not appear at all" help="(-r)" />
+
+         <param name="C" type="data" format="nhx" optional="true" label="Constraining tree" help="(-C)" />
```

Comment:
Add "in Newick format" to `label` or `help`. Also this line has 1 extra indentation space.